api/spec/src/app/stripe.tsp

```diff
@@ -0,0 +1,345 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/openapi3";
+
+import "./app.tsp";
+import "./capability.tsp";
+import "../errors.tsp";
+import "../oauth.tsp";
+import "../pagination.tsp";
+import "../types.tsp";
```

Comment:
If the app is imported in parent's main, I don't think this is necessary.

Reply:
Correct. It's currently not imported in `app/main.tsp`. Would it be preferred to import them there instead of per file?